currency.sol

```diff
@@ -239,6 +243,27 @@ contract cccur is owned {
   function balanceOf(address _from) public constant returns (int256 amount){
      return  balanceEL[_from] + balanceCM[_from];
   }
+
+  /* change the accountAlreadyUsed */
+  function use(address add) internal {
```

Comment:
It seems `owner` should also get `used` at `setFirstAdmin()` time, shouldn't it ?

Reply:
added in a new commit (also added on transfert ownership)